modin/data_management/factories.py

```diff
@@ -152,6 +153,25 @@ def read_sql(cls, **kwargs):
 
     @classmethod
     def _read_sql(cls, **kwargs):
+        if execution_engine != "Ray":
+            if "partition_column" in kwargs:
+                if kwargs["partition_column"] != None:  # noqa: E711
```

Comment:
I left a comment below about `is not` instead of `!=`.

Reply:
fixed